sycl/unittests/SYCL2020API/SYCL2020APITests.cpp

```diff
@@ -271,3 +271,36 @@ TEST(DefaultValues, DISABLED_DefaultValuesAreOverriden) {
   EXPECT_EQ(SpecConstVal0, 80);
   EXPECT_EQ(SpecConstVal1, 8);
 }
+
+TEST(KernelBundle, GetKernelBundleFromKernel) {
+  sycl::platform Plt{sycl::default_selector()};
+  if (Plt.is_host()) {
+    std::cerr << "Test is not supported on host, skipping\n";
```

Comment:
IIRC, some testing frameworks report fail if test application return success error code, but output anything to the error stream.
Do you think error stream is the right stream for this type of message?

Reply:
Doesn't look like a major issue. Is the suggestion is to change it to `out` or `log` or something else?

Reply:
`out` sounds good to me.